msal/src/Microsoft.Identity.Client/Platforms/iOS/UIParent.cs

```diff
@@ -62,5 +62,15 @@ public UIParent(bool useEmbeddedWebview) : this()
             CoreUIParent.UseEmbeddedWebview = useEmbeddedWebview;
         }
 
+        /// <summary>
+        /// Platform agnostic constructor that allows building an UIParent from a NetStandard assembly.
+        /// On iOS, the parent is ignored, you can pass null.
+        /// </summary>
+        /// <param name="parent">Ignored on iOS</param>
+        /// <param name="useEmbeddedWebview">Flag to determine between embedded vs system browser. See https://aka.ms/msal-net-uses-web-browser </param>
+        public UIParent(object parent, bool useEmbeddedWebview)         :
```

Comment:
On other platforms, we will require this exact constructor. I was thinking to enforce this with an interface, but ctors cannot be part of an interface. Instead, I could expose a builder object and add an interface to that. It would be more like "IUIParentBuilder" with methods "WithParent(object); UseEmbeddedWebview(bool); The downside is that the builder is less discoverable than a ctor.

Reply:
use a factory method to create if you need a specific interface for construction?